requirements_dev.txt

```diff
@@ -1,3 +1,6 @@
+django-sekizai
```

Comment:
Thinking we could add a `-r requirements.txt` here and in requirements_test.txt instead, should make it so we only got 1 spot to manage.